util/regexp-assemble/regexp-942180.data

```diff
@@ -1,3 +1,21 @@
+##! This is a data file used to generate a regular expression for a CRS rule. 
```

Comment:
The new result and the old rule differ. The changes should be reviewed and changed in the rule file if ok.

Reply:
Looks like more issues with escapes.

Reply:
After updating the script this regex is now identical except for an additional escape. I think that's ok.

Reply:
Same as above: the original script includes an additional backslash that is not present in the expression in the rule. Both script versions produce the same expression.